assets/javascripts/parking_site_map.js

```diff
@@ -321,16 +399,105 @@ window.addEventListener('DOMContentLoaded', () => {
       'REVERE',
       'WALTHAM',
       'WATERTOWN',
-      'WINTHROP'];
-    const filteredMunicipalities = data[0].features.filter(municipality => surveyedMunicipalities.includes(municipality.properties.town));
+      'WINTHROP',
+      'BEVERLY',
+      'CONCORD',
+      'DANVERS',
+      'LEXINGTON',
+      'SALEM',
+      'SUDBURY',
+      'NEEDHAM',
+      'PEABODY',
+      'LINCOLN',
+      'WAYLAND',
+      'WESTON',
+      'LYNNFIELD',
+      'WAKEFIELD',
+      'SAUGUS',
+      'LYNN',
+      'STONEHAM',
+      'WOBURN',
+      'WELLESLEY',
+      'SWAMPSCOTT',
+      'MARBLEHEAD',
+      'NAHANT',
+      'WINCHESTER'
+    ];
+    phases = ["1", "2", "3", "4"];
```

Comment:
Not anything to block a merge/deploy, but we could consider using a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) here. Not sure if order matters (maybe it does, skimming the rest of the code below), but if not, it's faster/easier to use `includes` with Set types. Probably easier to add/remove members of the set vs. Array splicing.

Reply:
Good point, it should be a quick fix. Let me do that right now